.github/workflows/CI.yml

```
@@ -40,6 +44,10 @@ jobs:
         strategy:
             max-parallel: 4
             fail-fast: false
+            matrix:
+              cxx: [g++-8, clang++]
+        env:
+          CXX: ${{ matrix.cxx }}
             
         steps:
         - uses: actions/checkout@v1
```

Beim Server wird hier die [zweite Version](https://github.com/actions/checkout) verwendet (vgl. [hier](https://github.com/SoPra-Team-17/Server/blob/ba9e1c770b8e8983ae92d2124c852c220f7cf52e/.github/workflows/CI.yml#L16)).
Spricht etwas dagegen das auch hier zu machen?
```suggestion
        - uses: actions/checkout@v2
```